substrate/client/cli/src/arg_enums.rs

```diff
@@ -278,6 +278,8 @@ pub enum SyncMode {
 	FastUnsafe,
 	/// Prove finality and download the latest state.
 	Warp,
+	/// minimize startup-time and disk space
```

Comment:
I think this doc comment is what appears to users in the `--help`. Worth mentioning that this option only works with RocksDB (unless we can make it work with ParityDB too...?)

```suggestion
	/// Minimize start-up time and disk space. (RocksDB only)
```